app/controllers/dashboard_controller.rb

```diff
@@ -1,9 +1,13 @@
 # frozen_string_literal: true
 
 require "velum/kubernetes"
+require "velum/suse_connect"
 
 # DashboardController shows the main page.
 class DashboardController < ApplicationController
+  skip_before_action :authenticate_user!, only: :autoyast
```

Comment:
As discussed in a previous meeting, this poses a security issue. Inside the autoyast file we have the @suse_regcode rendered and with no authentication/authorization in place anybody with network access to the dashboard can read the reg_code.

I think we should start planning for a solution. I think we can ignore this for now (beta1).